### ISSUE TITLE
Small mode

### ISSUE THREAD

**Issue by maddie-wang** [bibekg/react-schedule-selector#31]
can we have a small mode? small mode is very important to me because the scheduler is way too long on my chrome extension, and I can't change the height of the cells. its not nice for the user to have to scroll to fill out all the times

unfortunately even if i use renderDateCell to make each cell 15px height, it won't change the height of the cells. the cells seem to have a fix height of 25px. 

![image](https://user-images.githubusercontent.com/32725368/94299931-ec392280-ff2d-11ea-8303-20e3789db4bb.png)

**Comment by bibekg** [comment]
Thanks for filing the issue. It looks like the 25px fixed height is coming from the `TimeLabelCell` on the left.

![image](https://user-images.githubusercontent.com/8083680/97090553-77b8d880-15ec-11eb-985c-0de6467bf58a.png)

 I'll put together a fix for it soon but a short-term solution in the interim to unblock you may be to add this custom CSS (selecting any div with a class starting with `ScheduleSelector__TimeLabelCell`) and unsetting the height.

```css
div[class^=ScheduleSelector__TimeLabelCell] {
  height: 'unset' !important;
}
```

**Comment by bibekg** [comment]
Just published v3.0 which removes the fixed height for the time label and also introduces a `renderTimeLabel` prop that will optionally let you customize the time label exactly how you'd like.
